src/Compilers/CSharp/Portable/Compilation/SyntaxTreeSemanticModel.cs

```diff
@@ -736,28 +736,21 @@ internal override MemberSemanticModel GetMemberModel(SyntaxNode node)
 
                 switch (memberDecl.Kind())
                 {
-                    case SyntaxKind.DelegateDeclaration:
```

Comment:
I'm assuming we do have tests on delegate declaration, but this code was simply no longer needed?